core/trino-main/src/test/java/io/trino/execution/TestingPageSourceProvider.java

```diff
@@ -36,11 +36,6 @@
 public class TestingPageSourceProvider
         implements ConnectorPageSourceProvider
 {
-    public TestingPageSourceProvider()
-    {
-        System.out.println();
```

Comment:
I know we probably won't do this update, but I wanted to check if that'll work and are there going to be any gainst in job times.

I don't want to start bikeshedding, but are there any best practices about tests using logging or printing directly?

Reply:
i usually prefer logging to direct sout

Reply:
Note that Surefire itself doesn't like sout since it uses that for passing control messages and crashes depending on when a sout is done. 

Newer versions have this fixed though it seems but required some config - https://trinodb.slack.com/archives/CP1MUNEUX/p1649874615122919?thread_ts=1649865737.113819&cid=CP1MUNEUX

Reply:
> Newer versions have this fixed though it seems but required some config - https://stackoverflow.com/a/61936537/4594699

for posterity

- this references https://issues.apache.org/jira/browse/SUREFIRE-1614
- and suggests to use
   ```xm
   <plugin>
      <artifactId>maven-surefire-plugin</artifactId>
      <version>3.0.0-M5</version>
      <configuration>
        <!-- Activate the use of TCP to transmit events to the plugin -->
        <forkNode implementation="org.apache.maven.plugin.surefire.extensions.SurefireForkNodeFactory"/>
      </configuration>
    </plugin>
   ```

Reply:
There's a comment in the Surefire Jira issue that it is supposed to be fixed on master, so I guess we can wait for the next release. I'll close this PR until then.

Reply:
follow-up: downgrade surefire in airbase: https://github.com/airlift/airbase/pull/313